### DISCUSSION TITLE
Cannot run this project

### DISCUSSION THREAD

**Discussion by maicss** [danielroe/nuxt-vitest#54] [General]
Follow steps in readme `developemt` part:
- pnpm install
- pnpm dev:prepare
```
PS C:\Users\maicss\workspace\nuxt-vitest> pnpm dev:prepare

> @0.6.4 dev:prepare C:\Users\maicss\workspace\nuxt-vitest
> pnpm nuxi prepare playground

Nuxi 3.1.1

 ERROR  Cannot find module 'C:\Users\maicss\workspace\nuxt-vitest\playground\node_modules\vitest-environment-nuxt\dist\module.cjs'
```

What should I do?

**Comment by larsalbrecht** [comment]
Have the same problem (but im using npm)

just added nuxt-vitest like in the readme.

Then installation failed during nuxt prepare:

> nuxt prepare
>
> Nuxi 3.6.5  
> 
>  ERROR  Error while requiring module vitest-environment-nuxt/module: Error: Cannot find module '/path/to/project/vitest-environment-nuxt/module'                             09:32:40
> Require stack:
> - /path/to/project//index.js
> 
>  ERROR  Cannot find module '/path/to/project/vitest-environment-nuxt/module'                                                                                                 09:32:40
> Require stack:
> - /path/to/project/index.js
